index.js

```diff
@@ -1,69 +1,70 @@
-const input = document.querySelector('#input');
-const pokemonContainer = document.querySelector('.pokemon-container')
```

Comment:
It's nice when the variable declaration and assignments are close to where they are needed, to provide a context :)